core-bundle/src/Image/PictureFactory.php

```diff
@@ -31,6 +31,8 @@
  */
 class PictureFactory implements PictureFactoryInterface
 {
+    const ASPECT_RATIO_THRESHOLD = 0.05;
```

Comment:
@leofeyer can we mark this as `private` when merging upstream?

Reply:
Done.